src/sider/arbeidsuforhet/SendForhandsvarselSkjema.tsx

```diff
@@ -19,6 +19,7 @@ const texts = {
     "Skriv kort din vurdering av hvorfor vilkåret ikke er oppfylt.",
     "Hvis du har vurdert ordningen friskmelding til arbeidsformidling: skriv hvorfor ordningen ikke er aktuell og legg inn henvisning til §8-5.",
   ],
+  defaultTextareaValue: "Jeg vurderer å avslå sykepengene dine fordi ...",
```

Comment:
Foreslår `Vi vurderer` eller `NAV vurderer` i stedet for `Jeg vurderer`

Reply:
Stine tenker det burde være "jeg" 🤔

Reply:
<img src="https://media4.giphy.com/media/fXnKQqryaDkqKsLjen/giphy.gif"/>